docs/3_finetuner_in_action/3_1_text_to_text.md

```diff
@@ -88,7 +88,7 @@ Perhaps in the future, we might want to create another run where we experiment w
 
 ```{admonition} Backbones
 :class: tip
- Finetuner also supports fine-tuning other models, see the [Choose a backbone model]](../2_step_by_step/2_5_choose_back_bone.md) section.
+ Finetuner also supports fine-tuning other models, see the {ref}`choose backbone <choose-backbone>` section.
```

Comment:
Change this in `image_to_image` and `text_to_image` as well. These guides have other references too, so let's refactor all of them to be consistent.